webapp/src/assets/styles/mobility.scss

```diff
@@ -1,5 +1,7 @@
 @import "./theme";
 
+$overlay-padding-x-size: 48px;
```

Comment:
Why is this a variable? Why is this hard-coded to 48px and not based on spacing?